lyrionmusicserver/lyrionmusicserver.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Logitech Media Server Daemon
+Description=Lyrion Media Server Daemon
```

Comment:
Isn't it "Lyrion Music Server" , not "Lyrion Media Server"?

Reply:
Fixed [here](https://github.com/FabioLolix/PKGBUILD/pull/37).